modules/compute/src/integrationTest/java/org/apache/ignite/internal/compute/utils/InteractiveJobs.java

```diff
@@ -180,6 +192,24 @@ public String execute(JobExecutionContext context, Object... args) {
                 RUNNING_INTERACTIVE_JOBS_CNT.decrementAndGet();
             }
         }
+
+        /**
+         * If any of the args are strings, convert them to signals and offer them to the job.
+         *
+         * @param args Job args.
+         */
+        private static void offerArgsAsSignals(Object[] args) {
```

Comment:
Why not just send a signal from the test? Why should we pass signals to the job and this job will set signals to itself???